default.py

```diff
@@ -21,7 +21,7 @@
 import xbmcaddon
 
 # plugin constants
-_addon = xbmcaddon.Addon(id=sys.argv[0][9:-1])
+_addon = xbmcaddon.Addon(id=sys.argv[0][9:33])
```

Comment:
Might be an idea to make it static? If the name of the plugin is changed then it will break anyway, therefore we could also make it static, so it is not that easy to overlook? Just an idea, no biggie, though.

Reply:
Or use a regex extraction like here : https://github.com/smirgol/plugin.video.crunchyroll/pull/21/files#diff-d3a9878bb365d7659748c36fb58a5485eba5d44a86589d8b62f67345101f8c22R47